lib/Fetcher/CodeFetcher/Download.php

```diff
@@ -31,7 +31,7 @@ public function setup() {
       drush_set_option($name, $value);
     }
     if ($this->site['verbose']) {
-      $command = 'drush dl ' . implode(' ', $commandline_args) . ' --verbose';
+      $command = 'drush dl ' . implode(' ', $commandline_args) . '--cache=0 --verbose';
```

Comment:
Looks good except that `'--cache` won't leave a space after the last `$commandline_args` member is printed - probably leading to some invalid command building.